datasets/adv_glue/README.md

```diff
@@ -0,0 +1,56 @@
+---
+pretty_name: Adversarial GLUE
+languages:
+- en
+paperswithcode_id: adversarial-glue-a-multi-task-benchmark-for
+---
+
+# Dataset Card for "adv_glue"
+
```

Comment:
It looks like the table of content is missing here, feel free to copy it from the template here: https://github.com/huggingface/datasets/tree/master/templates

Note that every section should also appear in the dataset card, though it's ok if some of them are empty or simply contains `[More Information Needed]`